ariadne/contrib/sqlalchemy/objects.py

```
@@ -0,0 +1,124 @@
+    @staticmethod
+    def get_loader_registry_from_context(context: Any) -> LoaderRegistry:
+        """Get the `LoaderRegistry` from the GraphQL context.
+
+        Override this method to customize how the registry is retrieved.
+        """
+        try:
+            return context["loader_registry"]
+        except KeyError:
+            raise RuntimeError(
+                "LoaderRegistry not found in context under key 'loader_registry'"
+            )
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)